data/json/items/armor/misc.json

```diff
@@ -267,5 +267,27 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
+  },
+  {
+    "id": "shield_kite",
```

Comment:
That version of the kite shield that was [imported](https://github.com/CleverRaven/Cataclysm-DDA/pull/44879/files#diff-71279ee2b4d504e9901e283768c6094d463adaa5ae427e2ef057588daeaaaeee) is  outdated, I'd recommend importing from the mod author's [revised repository](https://github.com/chaosvolt/cdda_medieval_mod_reborn/blob/master/Medieval_Mod_Reborn/items/armor.json) instead. The updated version fixed some old oversights in the item's JSON not found in your imported version.

Reply:
Will update it soon, thanks.